conf/nomad/minio.hcl

```diff
@@ -8,6 +8,13 @@ job "${service_name}" {
     network {
       mode = "bridge"
     }
+
+    volume "persistence" {
+      type      = "host"
+      source    = "persistence"
```

Comment:
I think it should be input variable

Reply:
@zhenik : Good suggestion. Do we want to have **"persistence"** as an input variable to the minio module so that the user gets the flexibility to decide the directory/path where he would want to persist the data?